web/vtctld2/src/app/dashboard/shard.component.ts

```diff
@@ -318,16 +374,19 @@ export class ShardComponent implements OnInit, OnDestroy {
   }
 
   openIgnoreHealthErrorDialog() {
-    this.dialogSettings = new DialogSettings('Ignore', `Ignore Health Check for ${this.selectedTablet.label}`, '',
+    this.dialogSettings = new DialogSettings('Ignore', `Ignore Health Check for ${this.selectedTablet.label}`,
+                                             `Make ${this.selectedTablet.label} ignore certain health errors that match a pattern.`,
```

Comment:
Maybe describe the consequences in more detail?

e.g. talk about the fact that you can force a tablet with an error to be healthy and vtgate will serve from that tablet then